config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
```

Comment:
- [ ] AN error is received when a user tries to refresh the front end, the greetings don't display and this error shows on the API terminal, kindly fix this and ensure the app runs as expected.
![image](https://user-images.githubusercontent.com/94551484/228977903-0ec35be0-73d2-485d-a4ae-543d640cf110.png)